common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
```

Comment:
Instructor Dashboard mixin (referenced in other classes): [L45](https://github.com/edx/edx-platform/blob/027ef8bfa731089d5f91cb99c8a80851a2256e18/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L45)
Bulk Email: [L59](https://github.com/edx/edx-platform/blob/027ef8bfa731089d5f91cb99c8a80851a2256e18/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L59)
Auto-enroll: [L94](https://github.com/edx/edx-platform/blob/027ef8bfa731089d5f91cb99c8a80851a2256e18/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L94)
Track selection: [L275](https://github.com/edx/edx-platform/blob/027ef8bfa731089d5f91cb99c8a80851a2256e18/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L275)
Verification flow: [L281](https://github.com/edx/edx-platform/blob/027ef8bfa731089d5f91cb99c8a80851a2256e18/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L281)
Immediate verification flow: might be able to get rid of this one?
Upgrade: might be able to get rid of this one?
Fake payment: might be able to get rid of this one?
Proctored: might be able to get rid of this one?
...
Taking a look at the others, but do the first ones here make sense?